flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -1114,7 +1114,8 @@ class JobManager(
           Option(jobGraph.getSerializedExecutionConfig()
```

Comment:
I think the `FallbackRestartStrategy` could be set by default in the `ExecutionConfig`. Then we could get rid of the `Option` here.